schema_editor/app/scripts/views/boundary/list-controller.js

```diff
@@ -2,15 +2,16 @@
     'use strict';
 
     /* ngInject */
-    function BoundaryListController() {
+    function BoundaryListController(Boundaries) {
         var ctl = this;
         initialize();
 
         function initialize() {
+          ctl.bounds = Boundaries.query();
```

Comment:
Consider renaming `bounds` to `boundaries`, since it's a bit more explicit. The 4 extra characters gets minified away anyways!